components/BottomNavigation/examples/BottomNavigationTypicalUseExample.m

```diff
@@ -33,9 +33,11 @@ - (id)init {
   self = [super init];
   if (self) {
     self.title = @"Bottom Navigation";
-    _colorScheme =
+    MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
+    containerScheme.colorScheme =
```

Comment:
Tip: I'd prefer if the default container scheme values were kept.

Reply:
Done.